src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
Consider using SCSS variables for margin values to maintain consistency and reusability, especially if these values are used in multiple places or if they are related to the layout spacing convention used in the project.